src/blocks/test/e2e/global-setup.ts

```diff
@@ -0,0 +1,42 @@
+/**
+ * External dependencies
+ */
+import { request } from '@playwright/test';
+import type { FullConfig } from '@playwright/test';
+
+/**
+ * WordPress dependencies
+ */
+import { RequestUtils } from '@wordpress/e2e-test-utils-playwright';
+
+async function globalSetup( config: FullConfig ) {
+	const { storageState, baseURL } = config.projects[ 0 ].use;
+	const storageStatePath =
+		'string' === typeof storageState ? storageState : undefined;
+
+	console.log( `[INFO] Base URL Test Instance: ${baseURL}` );
+
+	const requestContext = await request.newContext({
+		baseURL
+	});
+
+	const r = await requestContext.head( baseURL );
+
+	console.log( '[Info] Request Status', r.status() );
+	console.log( '[INFO] Endpoint Link', r.headers().link );
```

Comment:
@HardeepAsrani, setting up the WP docker testing environment can be tricky.

It would be best to ensure you always have an updated docker image of it. I put those logs to show if the connection can be made with the REST endpoints of the WP.